states/huntsman/parked.py

```diff
@@ -6,7 +6,7 @@ def on_enter(event_data):
 
     pocs.say("Cleaning up for the night!")
 
-    if pocs.is_dark(horizon="flat") and not pocs.is_weather_safe():
+    if pocs.is_dark() and not pocs.is_weather_safe():
         pocs.next_state = 'taking_darks'
```

Comment:
O.K. so we are using the state `taking_darks` to actually take the darks. The problem is I can't see this state listed in the state table (`resources/state_table/huntsman.yaml`), which needs to declare the specify the allowable transitions in/out of `taking_darks`.